templates/documents/general_list.html

```diff
@@ -49,41 +49,39 @@
 </head>
 <!-- ViSolve - Call expandAll function on loading of the page if global value 'expand_document' is set -->
 {if $GLOBALS.expand_document_tree}
-  <body class="body_top" onload="javascript:objTreeMenu_1.expandAll();return false;">
+  <body onload="javascript:objTreeMenu_1.expandAll();return false;">
 {else}
-  <body class="body_top">
+  <body>
 {/if}
-<div class="container-fluid">
+<div class="container-fluid mt-3">
 	<div class="row">
 		<div class="col-sm-12">
 			<div class="title">
-					<h2>{xlt t='Documents'} <a href='interface/patient_file/summary/demographics.php' onclick='top.restoreSession()' title="{xla t='Go Back'}" ><i id='advanced-tooltip' class='fa fa-undo fa-2x small' aria-hidden='true'></i></a></h2>
+				<h2>{xlt t='Documents'} <a href='interface/patient_file/summary/demographics.php' onclick='top.restoreSession()' title="{xla t='Go Back'}" ><i id='advanced-tooltip' class='fa fa-undo fa-2x small' aria-hidden='true'></i></a></h2>
 			</div>
 		</div>
 	</div>
 	<div class="row">
 		<div class="col-sm-3">
 			<div id="documents_list">
 				<fieldset>
-				<legend>{xlt t='Documents List'}</legend>
-                <div style="padding: 0 10px">
-                    <div class="ui-widget form-inline" class='float-right'>
-                         <select id="selectPatient" class="form-control" type="text" data-placeholder="{$place_hld|attr}" style="width:150px;">
-                            <option></option></select>
-                        <button id='pid' class="btn btn-primary pBtn" type="button" class='float-right' style="border-radius: 0 0.25rem 0.25rem 0">0</button>
+                    <legend>{xlt t='Documents List'}</legend>
+                    <div class="pl-3">
+                        <div class="form-inline float-right" id="patientSearch">
+                            <select id="selectPatient" class="form-control" type="text" data-placeholder="{$place_hld|attr}" style="width:150px;">
+                                <option></option>
+                            </select>
+                            <button id='pid' type="button" class='float-right btn btn-primary pBtn' style="border-radius: 0 0.25rem 0.25rem 0">0</button>
```

Comment:
Can't believe I caught this one: Replace the 0 with `&times;` for the text inside the button. Also remove the border-radius inline style here.